docs/06-concepts/11-authentication/02-basics.md

```diff
@@ -96,3 +96,31 @@ class MyEndpoint extends Endpoint {
 :::caution
 Keep in mind that a scope is merely an arbitrary string and can be written in any format you prefer. However, it's crucial to use unique strings for each scope, as duplicated scope strings may lead to unintentional data exposure.
 :::
+
+## Managing Authentication in Serverpod's `StatusEndpoint`
+
+The `StatusEndpoint` class includes methods for handling user sign-outs, whether from a single device, all devices, or via a deprecated method.
+
+- **Sign out from the current device**
+
+    Use the `signOutDevice` method to log the user out of the current session only, leaving other sessions active.
```

Comment:
Question: We are using session here, is that inline with other documentation on this page or why are we switching from device to session?

Reply:
Fixed